src/instructlab/sdg/generate_data.py

```diff
@@ -264,29 +192,48 @@ def generate_data(
     """
     generate_start = time.time()
 
+    knowledge_recipe = Recipe(
+        "src/instructlab/sdg/configs/knowledge/data_recipe/default_recipe.yaml"
+    )
+    skills_recipe = Recipe(
+        "src/instructlab/sdg/configs/skills/data_recipe/default_recipe.yaml"
```

Comment:
I feel like the main purpose of this file is to specify some initial data sets that should be mixed with the generated datasets?

Couldn't that be specified in the pipeline config:

```
version: "1.0"
initial_datasets:
  - path: ../../datasets/mydataset.jsonl
    sampling_size: 30
blocks:
  - name: gen_questions
    type: LLMBlock
```

?

It also seems to serve a secondary purpose of writing a record of the datasets that were mixed to produce the final output?

Reply:
I split the concerns of figuring out how to inject initial data sets into #171, which means for the scope of this PR the default recipes are really just bringing the system prompt. These files don't seem like a great way to manage that, since as you pointed out they have to be equal in both knowledge and skills recipes. I'm giving a chance for some of the others I tagged to speak up here, but if they don't then will proceed with removing the default recipe files entirely from this PR for now.

Reply:
I split writing recipe files to disk out as #185 to reduce scope of this PR.